ipam/store.go

```
@@ -39,6 +39,12 @@ func (a *Allocator) Value() []byte {
 	return b
 }
 
+// SetValue unmarshalls the data from the KV store.
+func (a *Allocator) SetValue(value []byte) error {
+	a.subnets = byteArrayToSubnets(value)
```

please do not discard the error